### DISCUSSION TITLE
Unable to Verify my Student Status

### DISCUSSION THREAD

**Discussion by HackerGwen** [community/community#44253] [GitHub Education]
I have updated my github profile but still I am not able to re-verify my Student Benefits.
Can you please help me or recheck if any important detail is missing

[https://github.com/settings/profile](url)

![Screenshot (503)](https://user-images.githubusercontent.com/98099005/212397195-35db4be3-d7fc-494c-a920-8d76d4196944.png)

**Comment by samus-aran** [comment]
Hi @HackerGwen 👋 

Often this common pre-processing warning is displayed because the application detects differences between your profile information and the evidence you’ve supplied. We do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification process looks at your GitHub username, email, and name (as well as other points) to cross-reference the evidence submitted, so be sure that they match to ensure a smoother process.

Let us know how you get on 👍
